### ISSUE TITLE
Node.js error

### ISSUE THREAD

**Issue by krizzziz** [oznu/docker-homebridge#443]
### Describe The Bug

The container with Homebridge is failing to startup. I received a RPI4 and downloaded a Raspan Bullseye (latest image of the website). Installed Docker and Portainer. But now I am getting error messages that has to do something with node.js and now the containers does not startup.

### Docker Config

```yml
version: '2'
services:
  homebridge:
    image: oznu/homebridge:raspberry-pi
    restart: always
    network_mode: host
    volumes:
      - ./config:/homebridge
    environment:
      - PGID=1000
      - PUID=1000
      - HOMEBRIDGE_CONFIG_UI=1
      - HOMEBRIDGE_CONFIG_UI_PORT=8581
```

### Logs

```text
`Node.js[7152]: ../src/util.cc:188:double node::GetCurrentTimeInMicroseconds(): Assertion `(0) == (uv_gettimeofday(&tv))' failed.
Aborted (core dumped)`
```

### Host Operating System

Raspberry Pi OS / Raspbian

### Host Architecture

armv7l

**Comment by oznu** [comment]
See https://github.com/oznu/docker-homebridge/issues/441#issuecomment-1132242621

**Comment by krizzziz** [comment]
When trying`sudo apt install -t buster-backports libseccomp2` from #441. The output I get is this:
'Reading package lists... Done
E: The value 'buster-backports' is invalid for APT::Default-Release as such a release is not available in the sources' so the problem was not solved.
Also when looking into #441 it seems to have effect on older raspian installations. I downloaded mine today.

**Comment by Anghellos** [comment]
> When trying`sudo apt install -t buster-backports libseccomp2` from #441. The output I get is this:
> 
> ```
> Reading package lists... Done
> E: The value 'buster-backports' is invalid for APT::Default-Release as such a release is not available in the sources
> ```
> 
> so the problem was not solved. Also when looking into #441 it seems to have effect on older raspian installations. I downloaded mine today.

i tried to install homebridge today and i get the same issue. no solution yet

**Comment by krizzziz** [comment]
I did had the same issue when installing another docker (Scrypted) and I also reported this bug there. It was mentioned that it is Buster causing the issue. That I did not have a Bullseye Raspian. I did not understood because I downloaded the latest image from the raspberry website.
But after installing everything again (after downloading again latest Bullseye image) and installing Docker and Homebridge-docker again. The problem was not there and everything is working. So it seems that because I was using Buster I was having this issue all along.
